src/webpackPages.js

```diff
@@ -72,16 +73,34 @@ const webpackPages = (globalOptions) => {
     }
 
     const finishAll = () => {
+      if (typeof globalOptions.webpack === 'function') globalOptions.webpack = globalOptions.webpack(globalOptions)
+      if (!outputFiles || Object.keys(outputFiles).length < 1) {
+        rm(path.join(metalsmith._directory, '_tempOutput'), () => { })
+        const webpackError = 'No outputFiles for webpack'
+        console.log(webpackError)
```

Comment:
Logging?

Reply:
It may be good to keep this in for tracking when there is an error here.